src/common/options/global.yaml.in

```
@@ -5437,6 +5437,16 @@ options:
   flags:
   - runtime
   with_legacy: true
+- name: bluestore_zero_block_detection
+  type: bool
+  level: dev
+  desc: punch holes instead of writing zeros
+  long_desc: Intended for scale testing in Teuthology. Currently this is implemented with punch hole semantics,
```

Some users may not be aware of what teuthology is.  We happen to use it a lot but it's not required to exercise this:
```suggestion
  long_desc: Intended for large-scale synthetic testing. Currently this is implemented with punch hole semantics,
```